### PR TITLE
[RFC] move response, operations, events, to their own packages

### PR DESCRIPTION
**Pull request by tych0**
The idea here is to start the process of decoupling basic things in the lxd/ directory, to hopefully not have it be such a snarl of interdependant crap in the future.

Take a look and let me know what you think. I still can't decide if this is a good way forward or not.

### CODE REVIEW

File: lxd/container_lxc.go [hunk 1]
```diff
@@ -760,7 +762,7 @@ func (c *containerLXC) initLXC() error {
 		"/sys/firmware/efi/efivars",
 		"/sys/fs/fuse/connections",
 		"/sys/fs/pstore",
-		"/sys/kernel/debug",
+		"/sys/kernel/state.Debug",
```
Comment (stgraber) [inline lxd/container_lxc.go]: oops :)

**Comment by tych0** [comment]
On Fri, Dec 16, 2016 at 12:10:15PM -0800, Stéphane Graber wrote:
> stgraber commented on this pull request.
> 
> 
> 
> > @@ -760,7 +762,7 @@ func (c *containerLXC) initLXC() error {
>  		"/sys/firmware/efi/efivars",
>  		"/sys/fs/fuse/connections",
>  		"/sys/fs/pstore",
> -		"/sys/kernel/debug",
> +		"/sys/kernel/state.Debug",
> 
> oops :)

heh, sed gone wild :)

**Comment by stgraber** [comment]
For the record (since Jenkins archives results). Jenkins is showing fd leakage with this branch (sockets).

**Comment by tych0** [comment]
Closing this, as it's now painful to rebase.

**Comment by stgraber** [comment]
Sorry about that

**Comment by tych0** [comment]
On Fri, Dec 23, 2016 at 09:29:28AM -0800, Stéphane Graber wrote:
> Sorry about that

no worries, it was bound to happen :)

**Pull request closed without merging**